storage/database/badger_database.go

```diff
@@ -206,21 +205,28 @@ type badgerBatch struct {
 // Put inserts the given value into the batch for later committing.
 func (b *badgerBatch) Put(key, value []byte) error {
 	err := b.txn.Set(key, value)
+	if err == badger.ErrTxnTooBig {
```

Comment:
Shouldn't this error be handle in `badgerDB.Put()` and `badgerDB.Delete()`, too?

Reply:
I haven't handle this error in `badgerDB.Put()` and `badgerDB.Delete()`, because i think this error cannot be came out  from single(non-batch) operation. If this came out in there, maybe we need to adjust the badgerDB option related to transaction buffer size or something else.